src/serac/physics/utilities/functional/tests/functional_comparisons_cuda.cu

```diff
@@ -414,6 +426,11 @@ int main(int argc, char* argv[])
 {
   ::testing::InitGoogleTest(&argc, argv);
 
+  cudaDeviceSynchronize();
+  auto [functional_free_memory, functional_total_memory] = serac::accelerator::getCUDAMemInfo();
+  std::cout << "Initial: Free memory:" << functional_free_memory << " Total_memory:" << functional_total_memory
+            << std::endl;
```

Comment:
Do you think it's worth having an additional convenience function that returns a string so you could do something like:
```cpp
SLIC_INFO("Initial: " << serac::accelerator::getCUDAMemInfoString());
```

Reply:
I think that's a good idea. I can add a `<<` overload.

Reply:
you have INFO, WARNING, ERROR.  Only ERROR halts.

Reply:
Just to clarify - this doesn't need to be a `struct` with an `op<<` overload - just a function that returns a string is sufficient.